packages/config/config/devices/0x015f/mh-s511.json

```diff
@@ -0,0 +1,357 @@
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Basic CC Integration Setting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Key1 Responds to Basic Set and Get. No unsolicited report to Lifeline",
+					"value": 0
+				},
+				{
+					"label": "Key1 Responds to Basic Set and Get. Key1 WILL send unsolicited report to Lifeline",
+					"value": 1
+				},
+				{
+					"label": "All Keys respond to Basic Set. Basic Get ignored. No unsolicited report to Lifeline",
+					"value": 2
+				},
+				{
+					"label": "All Keys respond to Basic Set. Basic Get ignored. All keys send unsolicited report to Lifeline",
+					"value": 3
+				}
+			]
+		},
+		"10": {
+			"label": "Key 1 Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Single click to switch on/off",
+					"value": 0
+				},
+				{
+					"label": "When turned on, auto off after 'Key On Duration' seconds",
+					"value": 1
+				},
+				{
+					"label": "When turned off, auto on after 'Key On Duration' seconds",
+					"value": 2
+				},
+				{
+					"label": "Hold >3secs turns on, when released, turns off",
+					"value": 3
+				},
+				{
+					"label": "Single click for on/off, Hold >3secs on, release turns off",
+					"value": 4
+				},
+				{
+					"label": "Momentary switch: hold key turns on, off once released",
+					"value": 5
+				},
+				{
+					"label": "Hold >3secs turns on, Hold >3secs turns off",
+					"value": 6
+				}
+			]
+		},
+		"14": {
+			"label": "Disable Local Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "All keys are valid for local control",
+					"value": 0
+				},
+				{
+					"label": "All keys are invalid for local control",
+					"value": 1
+				}
+			]
+		},
+		"15": {
+			"label": "Disable Remote Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "All keys are valid for remote (gateway) control",
+					"value": 0
+				},
+				{
+					"label": "All keys are invalid for remote (gateway) control",
+					"value": 1
+				}
+			]
+		},
+		"16": {
+			"label": "Scene Respond",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Scene Respond Disabled",
+					"value": 0
+				},
+				{
+					"label": "Key 1 responds, Keys 2 and 3 do not respond",
```

Comment:
Are there only three keys? If so, just make it "Key 1 responds," then "Key 2 responds" for the next option, etc. If only one is listed and other options list only other it is implicit that the others do nothing.

Reply:
That makes sense.    will do